chore: add widget examples

nikaaru opened the pull request:
# Summary

This PR adds widget examples to the existing `examples/ `directory, updates `eslint.config.mjs` to include an ignores: `['examples/**']` entry so ESLint skips that folder, and removes examples from the root `package.json` workspaces array to prevent its dependencies from inflating the root `node_modules`.



# How did you test this change?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B


# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

yeager-eren commented on package.json:
```diff
@@ -8,7 +8,6 @@
   },
   "workspaces": {
     "packages": [
-      "examples/*",
```
by doing this, `queue-manager-demo` is not a member of workspace so our publish script won't upgrade its dependencis any more, I guess you need to use `latset` vesrion there as well.

nikaaru replied:
I moved all the `@rango-dev` packages to the latest.

nikaaru commented on .gitignore:
```diff
@@ -41,4 +41,7 @@ yarn-error.log*
\ No newline at end of file
+.nx
+
+examples/**/node_modules
```
I deleted it.

nikaaru commented on .gitignore:
```diff
@@ -41,4 +41,7 @@ yarn-error.log*
+.nx
+
+examples/**/node_modules
+examples/**/build
```
I moved all the build examples to the `dist` folder and deleted both of them.

nikaaru commented:
@yeager-eren 

I moved `queue-manager-demo` under `queue-manager` directory ([chore: move queue-manager-demo to queue-manager directory](https://github.com/rango-exchange/rango-client/pull/1217/commits/809963ea11db0ac4694a60ffa5f5ba7be3ed6a61)) and restored the previous **`yarn.lock`**. When I removed `/examples` from the `workspaces`, Yarn recalculated **hoisting** and changed `yarn.lock`, which broke a few builds.

For example: `visx` `TooltipWithBounds` started failing due to mixed `@types/react`, and `@mysten/sui`’s `d.ts` began pointing to an unresolved subpath. Keeping the demo inside the queue-manager workspace restores the original hoisting and keeps `yarn.lock` stable, and avoids those regressions without touching root configs.

yeager-eren reviewed (approved):
lgtm
